pages/common/ssh-keygen.md

```diff
@@ -22,6 +22,10 @@
 
 `ssh-keygen -l -F {{remote_host}}`
 
+- Retrieve the fingerprint of a key in MD5 Hex (useful for checking a key against the list in Github):
```

Comment:
I think this would be the last one before starting to swap new samples with more useful ones. We can deal with that separately as additions come.